docs/src/main/paradox/release-notes/releases-1.1.md

```diff
@@ -43,6 +43,7 @@ The Stream API has been updated to add some extra functions.
 * added extra retry operators that allow users to provide a predicate to decide whether to retry based on the exception ([PR1269](https://github.com/apache/pekko/pull/1269))
 * add optionalVia/unsafeOptionalDataVia operators ([PR1422](https://github.com/apache/pekko/pull/1422))
 * add alsoTo/alsoToContext operators to `SourceWithContext`/`FlowWithContext` ([PR-1443](https://github.com/apache/pekko/pull/1443))
+* add wireTap/wireTapContext operators to `SourceWithContext`/`FlowWithContext` ([PR-1446](https://github.com/apache/pekko/pull/1446))
```

Comment:
This is ok but I think when we put together the release notes, we should have separate sections for 1.1.0 and 1.1.0-M1 releases. We can consolidate them again later but until we get all the modules like HTTP and Connectors 1.1.0 releases, it is useful for 1.1.0-M1 users to know which changes are in that release.
We don't need to repeat the changes. 1.1.0 users can be told that that release contains everything in 1.1.0 plus the extra listed changes.

Reply:
Personally I think this process is a bit heavy and not giving much benefit considering that we are following SemVer it shouldn't make a difference about whether something is in M1 or in M2 or the first milestone.

We can discuss this on the mailing list if you want to push for it.

Reply:
The alternative is that we hijack the 1.1.0-M1 section of the release notes (where you have added this change) - and just annotate the additional changes that were not in 1.1.0-M1.

Reply:
After thinking about it I have just moved the 3 added api's since M1 to the appropriate section